aait/backend/group-2A/Domain/Entities/Comment.cs

```diff
@@ -8,8 +8,8 @@ public class Comment
         //The User That Commented
         public required int UserId { get; set; }
         public required int PostId { get; set; }
-        public required int CreatedAt{ get; set; }
-        public required int UpdatedAt{ get; set; }
+        public required DateTime CreatedAt{ get; set; }
+        public required DateTime UpdatedAt{ get; set; }
```

Comment:
create a baseEntity and add CreatedAt and DateTime to it. extend comment from baseEntity